e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/dashboard-deals-filter-created-by-you.spec.js

```diff
@@ -165,6 +168,9 @@ context('Dashboard Deals filters - Created by you', () => {
   });
 
   it('should be able to select multiple filters and remove created by you filter and list relevant deals', () => {
+    cy.login(BANK1_MAKER1);
+    dashboardDeals.visit();
```

Comment:
No need for `dashboardDeals.visit();` as login redirects to the dashboard.

Reply:
Removed